chapters/ch55.md

```diff
@@ -1,78 +1,82 @@
 
 ![img](../imgs/C55_01.png)
 
 > For each mistake the system makes on the dev set:
 
---> _replace THIS LINE by your translation for the above line_
+Với từng lỗi gây ra khi hệ thống chạy trên tập phát triển, 
 
 > 1. Try manually modifying A’s output to be a "perfect" output (e.g., the "perfect" bounding box for the cat), and run the rest of the pipeline B, C on this output. If the algorithm now gives a correct output, then this shows that, if only A had given a better output, the overall algorithm’s output would have been correct; thus, you can attribute this error to component A. Otherwise, go on to Step 2.
 
---> _replace THIS LINE by your translation for the above line_
+1. Thử điều chỉnh bằng tay kết quả đầu ra ở A cho "hoàn hảo" (ví dụ, chỉnh sửa khung chứa hình mèo cho "hoàn hảo"), và sau đó tiến hành chạy thuật toán cho pipeline gồm có B và C với đầu ra "hoàn hảo" của A đã thao tác ở trên. Nếu thuật toán cho ra kết quả đầu ra cuối cùng chính xác, điều đó cho ta thấy rằng, thuật toán sẽ cho ra kết quả chính xác khi và chỉ khi việc xử lý thông tin ở công đoạn A cho kết quả đầu ra tốt. Vậy ta có thể quy lỗi là do việc xử lý ở công đoạn A. Ngược lại, ta sẽ kiểm chứng thêm ở bước 2. 
 
 > 2. Try manually modifying B’s output to be the "perfect" output for B. If the algorithm now gives a correct output, then attribute the error to component B. Otherwise, go on to Step 3.
 
---> _replace THIS LINE by your translation for the above line_
+2. Tương tụ, ta thử điều chỉnh bằng tay kết quả đầu ra ở công đoạn B cho "hoàn hảo". Nếu thuật toán cho ra kết quả đầu ra cuối cùng chính xác, ta có thể quy lỗi là do việc xử lý ở công đoạn B. Ngược lại, ta tiến hành bước 3.
 
 > 3. Attribute the error to component C.
 
---> _replace THIS LINE by your translation for the above line_
+3. Sau khi đã kiểm chứng nguyên nhân phát sinh lỗi không phải do A và B, ta có thể kết luận rằng lỗi gây ra do thành phần C. 
+
 
 > Let’s look at a more complex example:
 
---> _replace THIS LINE by your translation for the above line_
+Chúng ta hãy cùng tìm hiểu một ví dụ phức tạp hơn sau đây:
 
 ![img](../imgs/C55_02.png)
 
 > Your self-driving car uses this pipeline. How do you use error analysis by parts to decide which component(s) to focus on?
 
---> _replace THIS LINE by your translation for the above line_
+Xe hơi tự lái của bạn sử dụng hệ thống pipeline như trên. Bạn sẽ sử dụng kỹ thuật phân tích lỗi từng phần như thế nào để quyết định thành phần (hay nhóm thành phần) nào cần cải thiện?
 
 > You can map the three components to A, B, C as follows:<br/>
 > A: Detect cars<br/>
 > B: Detect pedestrians <br/>
 > C: Plan path for car
 
---> _replace THIS LINE by your translation for the above line_
+Bạn có thể gọi tên 3 thành phần trong hệ thống là A, B, C tương ứng với các chức năng như sau:
+A: nhận dạng xe
+B: nhận dạng người đi bộ
+C: xác định đường cho xe
```

Comment:
Ở trong hình vẽ, "Plan path for car" được dịch thành "hoạch đình đường đi cho xe", chúng ta nên thống nhất một cách dịch. #